### DISCUSSION TITLE
Wow why is it using java 11??

### DISCUSSION THREAD

**Discussion by nsmith1024** [journeyapps/zxing-android-embedded#677] [Q&A]
Wow why is it using java 11?? how many people have that??

**Comment by rkistner** [comment]
I don't see the issue here - JDK 11 is over 3 years old by now. And it's nothing specific to this library - it's a requirement of the latest Android Gradle plugin.

You also only need to use the latest gradle plugin if you want to use desugaring, and there are other options described in the readme.
